src/Synchronizer.js

```diff
@@ -88,11 +88,38 @@ module.exports = Class.extend({
          })
          .catch(function(err) {
             self._abortScanning = true;
-            console.log('ERROR: encountered an error while comparing tables', err, err.stack);
+            console.error('Encountered an error while comparing tables', err, err.stack);
          })
          .then(this._outputStats.bind(this));
    },
 
+   trackScanProgress: function(enabled, approxItems, f) {
+      var tick = approxItems > 100 ? 1 : 5,
+          currentPercent = 0,
+          nextProgressPercent = 0,
+          ret;
+
+
+      ret = f(function(currentItems) {
+         currentPercent = Math.round(currentItems / approxItems * 100.0);
+
+         if (currentPercent < 100 && currentPercent >= (nextProgressPercent + tick)) {
+            process.stdout.write('(' + currentPercent + '%)');
```

Comment:
what would you think about adding `process.stdout.cursorTo(0);` or `process.stdout.clearLine();` before this write so that the percentage doesn't get constantly appended?

Reply:
I actually don't really know what cursorTo(0); does.  Default output looks something like: https://gist.github.com/er1c/4925fdf05c9b07b232874cf51ed32540 with the "dots" thrown in.

Reply:
This small patch:

```
diff --git a/src/Synchronizer.js b/src/Synchronizer.js
index ea8f73d..d992828 100644
--- a/src/Synchronizer.js
+++ b/src/Synchronizer.js
@@ -104,6 +104,8 @@ module.exports = Class.extend({
          currentPercent = Math.round(currentItems / approxItems * 100.0);
 
          if (currentPercent < 100 && currentPercent >= (nextProgressPercent + tick)) {
+            process.stdout.clearLine();
+            process.stdout.cursorTo(0);
             process.stdout.write('(' + currentPercent + '%)');
```

Would make it look like this:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/171934/78512900-2fcc8400-7776-11ea-9315-50f515bea37e.gif)

Reply:
updated!

Reply:
how did you make that gif/screenshot?

Reply:
QuickTime to screen record, and then https://ezgif.com/video-to-gif/ to convert